src/screens/CreatePost/index.js

```diff
@@ -18,10 +19,9 @@ import {
   Text,
   View
 } from 'react-native';
-import {Image} from 'react-native-compressor';
-import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
 import {openSettings} from 'react-native-permissions';
 import {showMessage} from 'react-native-flash-message';
+import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed and `openSettings` from 'react-native-permissions' is now imported before `launchCamera, launchImageLibrary` from 'react-native-image-picker'. This should not affect the functionality but ensure that all these modules are used correctly in the code.
```diff
- import {Image} from 'react-native-compressor';
- import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
+ import {openSettings} from 'react-native-permissions';
+ import {showMessage} from 'react-native-flash-message';
+ import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->